config/src/main/java/com/typesafe/config/impl/SerializedConfigValue.java

```diff
@@ -42,6 +43,8 @@
     // this is the version used by Java serialization, if it increments it's
     // essentially an ABI break and bad
     private static final long serialVersionUID = 1L;
```

Comment:
The data format has changed (and this is ABI break). Shouldn't be it incremented?

Don't sure, because the new implementation is still compatible with old one for short strings.